data/json/items/melee/spears_and_polearms.json

```diff
@@ -547,7 +547,7 @@
     "longest_side": "180 cm",
     "bashing": 6,
     "cutting": 40,
-    "to_hit": { "grip": "weapon", "length": "long", "surface": "point", "balance": "uneven" },
+    "to_hit": { "grip": "weapon", "length": "long", "surface": "line", "balance": "neutral" },
```

Comment:
This shouldn't be as good as the properly forged naginata. It currently does *more* dps. I think it wouldn't be unreasonable to have it, have worse balance, it is just made of two pipes and some ductape.

Reply:
I still think it should be neutral balance, but you're right that the DPS shouldn't be higher than the "real" one.  I took it down a few points.